src/titiler/core/titiler/core/factory.py

```diff
@@ -824,189 +823,301 @@ def wmts(
-                    out_wms_params["dst_crs"] = value
-                    out_wms_params["bounds_crs"] = value
-                    
-                elif key == "bbox":
-                    value = [float(xy) for xy in value.split(",")]
-                    if len(value) != 4:
-                        raise HTTPException(
-                            status_code=400,
-                            detail="bbox needs 4 coordinates separated by commas",
-                        )
-                elif key == "version" and value not in set(WMSVersion):
-                    raise HTTPException(
-                        status_code=400,
-                        detail=f"Version specified not allowed. Allowed versions include: {[w.value for w in list(WMSVersion)]}",
-                    )
-                elif key == 'format':
-                    value = WMSMediaType(value)
-                    
-                if key == "layers":
-                    out_wms_params[key] += [value]
-                else:
-                    out_wms_params[key] = value
-                    
-            if out_wms_params.get("bbox"):
-
-                # WMS 1.3.0 is lame and flips the coords of EPSG:4326
-                if (
-                    out_wms_params.get("version") == "1.3.0"
-                    and out_wms_params.get("srs")
-                    and out_wms_params.get("srs") == "EPSG:4326"
-                ):
-                    out_wms_params["bbox"] = [
-                        out_wms_params["bbox"][1],
-                        out_wms_params["bbox"][0],
-                        out_wms_params["bbox"][3],
-                        out_wms_params["bbox"][2],
-                    ]
-                    
-                # Overwrite CRS:84 with EPSG:4326 when specified
-                elif (
-                    out_wms_params.get("version") == "1.3.0"
-                    and out_wms_params.get("srs") == "CRS:84"
-                ):
-                    out_wms_params["srs"] = "EPSG:4326"
-                    
-                # `bounds` is compatible with reading from rio.io.reader
-                out_wms_params['bounds'] = out_wms_params['bbox']
-                    
-            print(out_wms_params)
-            return out_wms_params
-
-        @self.router.get("/wms")
+        @self.router.get(
+            "/wms",
+            response_class=Response,
+            responses={
+                200: {
+                    "description": "Web Map Server responses",
+                    "content": {
+                        "application/xml": {},
+                        "image/png": {},
+                        "image/jpeg": {},
+                        "image/jpg": {},
+                        "image/webp": {},
+                        "image/jp2": {},
+                        "image/tiff; application=geotiff": {},
+                    },
+                },
+            },
+            openapi_extra={
+                "parameters": [
```

Comment:
👇 with openapi_extra we add entries to the openapi.json schema (swagger docs) but this is just for `display` 😄 (the api will accept lower and upper case)